parquet/file/column_reader.go

```diff
@@ -113,6 +113,8 @@ type ColumnChunkReader interface {
 	// automatically read the first page of the page reader passed in until
 	// HasNext which will read in the next page.
 	setPageReader(PageReader)
+	// Close releases the resources held by the column reader.
+	Close()
```

Comment:
if we're adding a `Close()` method to the reader we should follow the convention and the signature should be `Close() error`